catalog/ui/search-ui/standard/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -11,64 +11,103 @@
  * <http://www.gnu.org/licenses/lgpl.html>.
  *
  **/
-
  -->
 <metatype:MetaData xmlns:metatype="http://www.osgi.org/xmlns/metatype/v1.0.0">
 
     <OCD name="Standard Search UI" id="org.codice.ddf.ui.search.standard.properties">
-        <AD
-                description="Specifies the number of results to request from each source (Max 1000)"
-                name="Result Count" id="resultCount" required="true" type="Integer"
-                default="250" max="1000" min="1">
-        </AD>
-        <AD name="Imagery Providers" id="imageryProviders"
-            required="false" type="String" cardinality="10000"
-            description='List of imagery providers to use. Valid types are: OSM (OpenStreetMap), AGM (ArcGisMap), BM (BingMap), WMS (WebMapService), WMT (WebMapTile), TMS (TileMapService), GE (GoogleEarth). Example: {"type": "OSM", "url" "http://a.tile.openstreetmaps.org" "layers" ["layer1" "layer2"] "parameters" {"FORMAT" "image/png" "VERSION" "1.1.1"} "alpha" 0.5}'
-            default=''
-        />
-
-        <AD name="Terrain Provider" id="terrainProvider"
-            required="false" type="String"
-            description='Terrain provider to use for height data. Valid types are: CT (CesiumTerrain), AGS (ArcGisImageServer), VRW (VRTheWorld). Example: {"type" "CT" "url" "http://example.com"}'
+
+        <AD id="resultCount"
+            name="Result Count"
+            description="Specifies the number of results to request from each source (Max 1000)."
+            type="Integer"
+            default="250"
+            max="1000"
+            min="1"/>
+
+        <AD id="imageryProviders"
+            name="Imagery Providers"
+            description='List of imagery providers to use. Valid types are: OSM (OpenStreetMap), AGM (ArcGisMap), BM (BingMap), WMS (WebMapService), WMT (WebMapTile), TMS (TileMapService), GE (GoogleEarth). Example: {"name": "Example OSM", "show": true, "type": "OSM", "url": "http://a.tile.openstreetmap.org", "fileExtension": "png",  "order": 0, "alpha": 1.0 }.'
```

Comment:
There are no content changes in this file except for the example in this description was updated to match the example that was recently updated in the documentation: https://github.com/codice/ddf/commit/d935a855ea9676af76ec0cce32e272f249bf43bd#diff-7095940451c775dd1ea938eeffe3a024R32.